sources/lib/Converter/Type/Box.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/*
+ * This file is part of PommProject's Foundation package.
+ *
+ * (c) 2014 Grégoire HUBERT <hubert.greg@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PommProject\Foundation\Converter\Type;
+
+/**
+ * Box
+ *
+ * PHP type for PostgreSQL's box type.
+ *
+ * @package Foundation
+ * @copyright 2017 Grégoire HUBERT
+ * @author Miha Vrhovnik
+ * @license X11 {@link http://opensource.org/licenses/mit-license.php}
+ */
+class Box
+{
+    public $topX;
+    public $topY;
+    public $bottomX;
+    public $bottomY;
+
+    /**
+     * __construct
+     *
+     * Create a box from a string description.
+     *
+     * @access public
+     * @param  string $description
+     */
+    public function __construct($description)
+    {
+        $description = trim($description, ' ()');
+
+        if (!preg_match('/([0-9e\-+\.]+), *([0-9e\-+\.]+) *\), *\( *([0-9e\-+\.]+), *([0-9e\-+\.]+)/', $description, $matches)) {
```

Comment:
This line does not respect the coding conventions (PSR2), this is why it fails in Travis (php-cs inspection). You have to either split the call to `preg_match` or the store the regular expression in a separate variable.